.gitignore

```diff
@@ -1,61 +1,65 @@
-# Compiled Object files
-**.o
-**.lo
-
-# Compiled Dynamic libraries
-**.so*
-
-# Compiled Static libraries
-**.a
-**.la*
-
-# Executables
-**_x
-
-# Pyfhel building
-Pyfhel/*.pyc
-Pyfhel/*.pyd
-Pyfhel/*.lib
-Pyfhel/*.dll
-Pyfhel/*.exp
-Pyfhel/*.cpp
-examples/playground
-**/build
-**.eggs
-**.egg-info
-**/dist
-**/gmon.out
-**/venv
-
-# Coverage
-.cov
-.coverage
-.pytest_cache
-htmlcov
-coverage.xml
-
-# Makefiles
-**/Makefile
-
-# Pyfhel saved objects
-**.pycon
-**.pypk
-**.pysk
-**.pyrok
-**.pyrlk
-
-# Visual Studio Code
-**.vscode
-
-# ipython checkpoints
-**.ipynb
-**.ipynb_checkpoints
-**/__pycache__
-
-# Documentation
-docs/_autosummary
-docs/_autoexamples
-**.doctree
-**.pickle
-docs/html
-docs/doctrees
\ No newline at end of file
+# macOS
```

Comment:
It seems you're adding a single line but the entire file is being modified. Make sure only the new line/s are marked as modifications (the easiest way would be to add them at the end instead of the beginning).